usr/share/rear/prep/Linux-i386/370_detect_secure_boot.sh

```diff
@@ -0,0 +1,34 @@
+test "$SECURE_BOOT_BOOTLOADER" && return 0
+
+# Note:
+# this is more of a hack than a really good solution.
+# The good solution would check the EFI variables for the bootloader
```

Comment:
With this explanation all is perfectly fine, see
"Dirty hacks welcome" in
https://github.com/rear/rear/wiki/Coding-Style